### DISCUSSION TITLE
Validation Failed in the "Build a CRUD app - REST" tutorial

### DISCUSSION THREAD

**Discussion by vinicius3w** [Budibase/budibase#4559] [Help]
Hello everyone, I´m trying to follow the "Build a CRUD app - REST". Everything is going well until I reach the step "Add a “Create Item” request".

In this step, I follow the whole instructions as follow (I replaced the sensitive information with "XXXXXXX"):

the params specification:
![image](https://user-images.githubusercontent.com/229764/154595430-26d4da92-a1ce-4c44-b35c-333092ee3401.png)

the JSON body specification:
![image](https://user-images.githubusercontent.com/229764/154595534-4791cf75-d7e6-4c2e-85db-590764e11a44.png)

And I got an error (Validation Failed) as a return...
![image](https://user-images.githubusercontent.com/229764/154595591-021eaf86-3a7b-410d-a9c3-a2f78aaa0a3c.png)

**Comment by mjashanks** [comment]
Ah. Thanks @vinicius3w 

That tutorial is a bit out of date, since we updated the rest connector.

We renamed "Parameters" to "Bindings". So, input your `title` variable into the `bindings` section. `Parameters` now means "URL Parameters".

Thanks for pointing this out.

**Reply by vinicius3w** [reply]
Oh... now it´s working! :D
Thank you!
